packages/analytics-js/__tests__/components/userSessionManager/UserSessionManager.test.ts

```diff
@@ -203,6 +203,66 @@ describe('User session manager', () => {
       userSessionManager.syncStorageDataToState();
       expect(state.session.anonymousId.value).toBe('dummy-anonymousId');
     });
+    it('should not set sessionInfo if autoTrack is set to false in loadOption and sessionInfo exists in storage', () => {
```

Comment:
If we were to run this test case without the code changes in this PR, would it fail?

Reply:
Yes